models/interval.go

```diff
@@ -143,14 +143,19 @@ func (i Interval) Validate() (bool, error) {
 			}
 		}
 		if i.Frequency != "" {
+			/* legacy frequencyPattern */
 			matched, _ := regexp.MatchString(frequencyPattern, i.Frequency)
 			if matched {
 				if i.Frequency == "P" || i.Frequency == "PT" {
 					matched = false
 				}
 			}
 			if !matched {
-				return false, NewErrContractInvalid(fmt.Sprintf("invalid Interval Frequency %s", i.Frequency))
+				// parse frequency example "1h15m30s10us9ns"
+				_, err := time.ParseDuration(i.Frequency)
+				if err != nil {
+					return false, NewErrContractInvalid(fmt.Sprintf("invalid Interval dude frequency %s format", i.Frequency))
```

Comment:
`dude frequency`?

Reply:
I, too, am intrigued and will be following this thread 👍

Reply:
Sorry about that...good catch.  That was an attempt to debug the use of go-mod-contracts locally referenced with the modification of the go-mod-contracts in edgeX-go project locally.  I neglected to change the text back.